src/assets/scripts/aircraft/AircraftConflict.js

```diff
@@ -136,7 +136,7 @@ export default class AircraftConflict {
             const isWarning = true;
             window.uiController.ui_log(`${this.aircraft[0].getCallsign()} collided with ${this.aircraft[1].getCallsign()}`, isWarning);
 
-            prop.game.score.hit += 1;
+            window.gameController.events_recordNew('COLLISION');
```

Comment:
make sure to update these to use the new enumeration.  This should now be `GAME_EVENTS.COLLISION`.